common/types/bool_test.go

```diff
@@ -65,6 +65,15 @@ func TestBool_ConvertToNative_Json(t *testing.T) {
 	}
 }
 
+func TestBool_ConvertToNative_Ptr(t *testing.T) {
+	ptrType := true
+	refType := reflect.TypeOf(&ptrType)
+	val, err := True.ConvertToNative(refType)
+	if err != nil || IsError(val) || !*val.(*bool) {
```

Comment:
I found only bool checks IsError(val). Does other types need to also check IsError(val)?

Reply:
Ah, no, that was leftover from a previous test case that I'd copied. Removed.